optimade/filtertransformers/base_transformer.py

```diff
@@ -22,29 +67,101 @@ class BaseTransformer(abc.ABC, Transformer):
         "=": None,
     }
 
-    # map from back-end specific operators to their inverse
-    # e.g. {"$lt": "$gt"} for MongoDB.
-    reversed_operator_map: Dict[str, str] = {}
+    # map from operators to their syntactic (NOT logical) inverse to handle
```

Comment:
Perhaps we should write "Not" instead of "NOT", as "NOT"  may be confused with the logical operator NOT.

Reply:
```suggestion
    # map from operators to their syntactic (as opposed to logical) inverse to handle
```